lib/matplotlib/path.py

```diff
@@ -175,7 +179,7 @@ def _fast_from_codes_and_verts(cls, verts, codes, internals_from=None):
             never copied, and always set to ``False`` by this constructor.
         """
         pth = cls.__new__(cls)
-        pth._vertices = _to_unmasked_float_array(verts)
+        pth._vertices = verts
```

Comment:
Can you add a note to the docstring that `verts` should be unmasked, and not just any NumPy array?